src/mesh.cpp

```diff
@@ -407,6 +407,13 @@ scene::IMesh* cloneMesh(scene::IMesh *src_mesh)
 
 scene::IMesh* convertNodeboxNodeToMesh(ContentFeatures *f)
 {
+	std::vector<aabb3f> boxes = f->node_box.fixed;
+	return convertNodeboxesToMesh (boxes, NULL);
+}
+
+scene::IMesh* convertNodeboxesToMesh(std::vector<aabb3f> &boxes, f32 *txc)
```

Comment:
Pointers are supposed to be on the right side of the space in the type name, not the left.

boxes needs to be a pointer if you're modifying it.